README.md

```diff
@@ -5,17 +5,23 @@ Maintain deb and rpm repositories on s3
 ## Ubuntu, Debian
 Install dvc from dep repository:
 ```
-$ sudo cp deb/dvc.list /etc/apt/sources.list.d/
-$ sudo apt-key add iterative.asc
-$ sudo apt-get update
-$ sudo apt-get install dvc
+$ sudo wget \
+       https://dvc.org/deb/dvc.list \
+       -O /etc/apt/sources.list.d/dvc.list
+$ wget -qO - https://dvc.org/deb/iterative.asc | gpg --dearmor > packages.iterative.gpg
+$ sudo install -o root -g root -m 644 packages.iterative.gpg /etc/apt/trusted.gpg.d/
+$ rm -f packages.iterative.gpg
+$ sudo apt update
+$ sudo apt install dvc
 ```
 
 ## Fedora, Centos, RHEL
 Install dvc from rpm repository:
 ```
-$ sudo cp rpm/dvc.repo /etc/yum.repos.d/
+$ sudo wget \
+       https://dvc.org/rpm/dvc.repo \
+       -O /etc/yum.repos.d/dvc.repo
 $ sudo rpm --import https://dvc.org/rpm/iterative.asc
-$ sudo dnf update
-$ sudo dnf install dvc
+$ sudo yum update
+$ sudo yum install dvc
```

Comment:
[Q] Will this work on RHEL?

Reply:
yum is obsolete though, dnf is the new package manager. Unless I've missed something.

Reply:
Ah, I see dvc.org still uses it. I think this was from an attempt to be compatible with old rhels, but we can get rid of it at this point.

Reply:
ok, so docs are wrong for Fedora/CentOS and README is wrong for Debian/Ubuntu 🤦🏻